loaders/express.js

```diff
@@ -14,6 +15,7 @@ module.exports = (app) => {
 
   app.use(
     session({  
+      secret: SESSION_SECRET,
```

Comment:
trebalo je među ostalim dodat i ovaj session secret.. to je private key kojim oni kriptiraju sadrzaj cookiea koliko san skuzija na brzinu